### ISSUE TITLE
Does tensorflow have a detach method like in pytorch to interrupt backforward?

### ISSUE THREAD

**Issue by hanxi0511** [tensorflow/tensorflow#56879]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Support

### Source

binary

### Tensorflow Version

2.9.1

### Custom Code

Yes

### Current Behaviour?

```shell
tell me the Method to interrupt gradient propagation
```

### Standalone code to reproduce the issue

```shell
nothing
```

### Relevant log output

_No response_</details>

**Comment by sushreebarsa** [comment]
@hanxi0511 Could you provide more details on the issue reported? Please refer this [link ](https://www.tensorflow.org/guide/autodiff)and let us know if it helps? 
Thank you!
